plugins/VisitsSummary/Controller.php

```diff
@@ -106,6 +107,17 @@ public function getEvolutionGraph(array $columns = array(), array $defaultColumn
             'avg_time_generation'
         );
 
+        $period = Common::getRequestVar('period', false);
+
+        if (SettingsPiwik::isUniqueVisitorsEnabled($period)) {
+            // add number of unique visitors if available
+            $selectableColumns = array_merge(
+                array($selectableColumns[0]),
+                array('nb_uniq_visitors'),
+                array_slice($selectableColumns, 1)
+            );
```

Comment:
In Piwik 3 there is already a sparklines visualization and it is defined very differently. See eg https://github.com/piwik/piwik/blob/3.x-dev/plugins/VisitsSummary/Reports/Get.php#L164